privacy_evaluator/metrics/basics.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+
+def accuracy(y: np.ndarray, y_prediction: np.ndarray) -> np.float32:
```

Comment:
this function would definitely need an assertion of the shape of the arrays that come in. otherwise, it might compute garbage